tar/write.c

```diff
@@ -1055,3 +1070,190 @@ test_for_append(struct bsdtar *bsdtar)
 */
 
 }
+
+static int
+bsdtar_write_open_fd(struct bsdtar *b, struct archive *a, int fd)
```

Comment:
Does this need to be duplicated in bsdtar?  I thought `write_open_fd` was only used for stdout, which shouldn't need any multivolume support.